README.md

```diff
@@ -1,13 +1,15 @@
 # vscode-cpptools
 
-This repo is the official repository for filing issues against and getting support for the Microsoft C/C++ extension for VsCode (https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools).
+This is the official repository for filing issues against and getting support for the [Microsoft C/C++ extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools).
 
 ### Microsoft Open Source Code of Conduct
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments. 
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact opencode@microsoft.com with any additional questions or comments. 
```

Comment:
any particular reason why the mailto: link is removed?

Reply:
The link seems to work without it? Maybe GitHub is smart about it and adds stuff.

Reply:
It was redundant. A mailto link is automatically generated for an email address.